src/melee/gm/gmmain_lib.c

```diff
@@ -5,6 +5,70 @@
 #include "lb/lbtime.h"
 
 #include <dolphin/os/OSReset.h>
+#include <sysdolphin/baselib/random.h>
+#include <sysdolphin/baselib/video.h>
+#include <melee/gm/gm_1601.h>
+#include <melee/gm/types.h>
+#include <melee/lb/lb_00B0.h>
+#include <melee/lb/lbaudio_ax.h>
+#include <melee/lb/lblanguage.h>
+
+GameRules gmMainLib_803D4A48 = {
+    0,
+    0x34,
+    0,  // mode
+    2,  // time limit
+    3,  // stock count
+    0,  // handicap
+    10, // damage ratio
+    0,
+    0,     // stock time limit
+    false, // friendly fire
+    true,  // pause
+    0,     // score display
```

Comment:
Is there a better way to do this in C?

Reply:
do what?

Reply:
The initializer comments.

Reply:
If we were using a compiler that wasn't from the stone age I'd use [designated initializers](https://gcc.gnu.org/onlinedocs/gcc/Designated-Inits.html) but MWCC doesn't support those :smiling_face_with_tear: